defaults/genesis.json

```diff
@@ -149,6 +149,65 @@
           "destination": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
         }
       }
+    },
+    {
+      "Grant": {
+        "Permission": {
+          "object": {
+            "name": "CanMintAssetWithDefinition",
+            "payload": {
+              "asset_definition": "rose#wonderland"
+            }
+          },
+          "destination": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
+        }
+      }
+    },
+    {
+      "Grant": {
+        "Permission": {
+          "object": {
+            "name": "CanManagePeers",
+            "payload": null
+          },
+          "destination": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
```

Comment:
Alice's private key is actually public for now, so this key pair needs to be replaced in secret. Otherwise anyone can act as Alice and un/register peers, reset parameters, un/register domains, update to arbitrary executor, etc. @BAStos525

Reply:
Yes, but Alice's public key should be consistent with genesis. If you share it with me I'll update genesis in this PR

Reply:
Just `ACCOUNT_PUBLIC_KEY` and `ACCOUNT_PRIVATE_KEY` env vars export?

Reply:
Let's think of it as replacing Alice with another administrator, not replacing Alice's keys. Done by #5284